RUNCTRL/iocBoot/iocRUNCTRL_01/st.cmd

```diff
@@ -33,12 +33,31 @@ dbLoadRecords("$(TOP)/db/runcontrolMgr.db","P=$(MYPVPREFIX),ALERT_OUT=$(MYPVPREF
 ## load run control settings written by blockserver
 iocshLoad "${ICPCONFIGROOT}/rc_settings.cmd", "RUNCONTROL=$(TOP)"
 
-## load LOQ specific detector control, this puts in the aperture if the detctor count rate is exceeded
-stringiftest("LOQ", "$(ICPCONFIGHOST)", 5, "NDXLOQ")
-$(IFLOQ) dbLoadRecords("$(TOP)/db/LOQ_detector.db","P=$(MYPVPREFIX)")
-$(IFLOQ) dbLoadRecords("$(WEBGET)/db/sendAlert.db","P=$(MYPVPREFIX),Q=CS:DC:ALERTS:,INST=$(INSTRUMENT=Unknown),SOURCE=IBEX")
-$(IFLOQ) dbLoadRecords("$(RUNCONTROL)/db/gencontrolMgr.db","P=$(MYPVPREFIX),MODE=DC,OUT_ACTION=$(MYPVPREFIX)CS:OVERCOUNT:ALERT.PROC")
-$(IFLOQ) dbLoadRecords("$(RUNCONTROL)/db/gencontrol.db","P=$(MYPVPREFIX),MODE=DC,PV=$(MYPVPREFIX)DAE:AD1:INTG:SPEC:RATE,NOALIAS=#")
+## load detector control, this puts in the aperture if the detctor count rate is exceeded
+stringiftest("DETECT", "$(ICPCONFIGHOST)", 5, "NDXLOQ")
```

Comment:
Everything looks good to me, just a typo here `...aperture if the detector`

Reply:
This was updated to be more descriptive